migrations/20180508154706_add_user_settings_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddUserSettingsTable extends AbstractMigration
+{
+    /**
+    * Migrate Up.
+    */
+    public function up()
+    {
+        $this->table('user_settings')
+            ->addColumn('config_key', 'string', [
```

Comment:
Can you add an index since we'll be searching by config key?

Reply:
Yep I have it's just not committed just yet.